scss/sidebar.scss

```diff
@@ -1,7 +1,102 @@
 .discourse-sidebar .d-toc-regular .d-toc-article .row {
   width: 75%;
-  @media screen and (max-width: 1200px) {
-    width: 65%;
+  @media screen and (max-width: calc($large-width + 100px)) {
+    width: calc(100vw - 240px - 3em);
+  }
+}
+
+@media screen and (max-width: calc($large-width + 100px)) {
+  .discourse-sidebar {
+    .d-toc-regular {
+      .post-bottom-wrapper {
+        padding: 1em 0.75em;
+        &.desktop {
+          display: none;
+        }
+      }
+      #d-toc {
+        z-index: z("header") + 1;
+        background: var(--secondary, $secondary);
```

Comment:
I don't think those double value vars are necessary? `var(--secondary)` should be enough?